### PR TITLE
Auto scroll toggle

### PR DESCRIPTION
**Pull request by mntxca**

### CODE REVIEW

**Comment by dsx1123** [comment]
Prefer to avoid any more buttons on create page, we have 4 buttons before, now we have 8, will it be possible to stop auto-scroll when the scroll bar is not at the bottom, restart the auto-scroll when the scroll bar is back to the bottom?

File: terraform/static/assets/cui-2.0.0/js/functionLibrary.js [hunk 1]
```diff
@@ -137,3 +137,42 @@ function loadInputLimit(limitArr) {
...
+
+function endAutoScroll() {
+  if (autoScrollTimer !== null)
+    clearInterval(autoScrollTimer);
+}
+
+let autoScrollBool = true;
+function toggleAutoScroll() {
+  autoScrollBool = !autoScrollBool;
+  autoScrollButton = document.getElementById("autoScrollToggle");
```
Comment (samiib) [inline terraform/static/assets/cui-2.0.0/js/functionLibrary.js:169]: `const autoScrollButton `
Best practice is to always declare variables with var, let or const.
You may have just missed this one.  😸

**Review by samiib — CHANGES_REQUESTED** [review]
Agree with @dsx1123 about the auto scroll. If you can implement it, would be awesome. :)

File: terraform/static/assets/cui-2.0.0/js/functionLibrary.js (hunk 1 above)
Comment (mntxca) [reply terraform/static/assets/cui-2.0.0/js/functionLibrary.js:169]: Nice catch! Will fix that

**Comment by mntxca** [comment]
> Prefer to avoid any more buttons on create page, we have 4 buttons before, now we have 8, will it be possible to stop auto-scroll when the scroll bar is not at the bottom, restart the auto-scroll when the scroll bar is back to the bottom?

I was originally considering that, but was unsure if that would be intuitive for a first time user of the ui. The user would have to try and see that feature exists instead of being able to see a button that clearly indicates the feature exists and can be toggled. Thus, that was the reasoning I went with having a button. I will work on creating the scroll bar at the bottom implementation, as I do agree we already have a lot of buttons on the page.

File: terraform/static/assets/cui-2.0.0/js/functionLibrary.js (hunk 1 above)
Comment (mntxca) [reply terraform/static/assets/cui-2.0.0/js/functionLibrary.js]: I added const there, is fixed now in latest commit

**Pull request merged**